js/app/Visualization/Animation/AnimationManager.js

```diff
@@ -306,6 +306,26 @@ function(Class,
       );
     },
 
+    getReportableAnimation: function() {
+      var self = this;
+
+      var predicate = function(animation) {
+        var hasSourceUrl =
+          animation.args &&
+          animation.args.source &&
+          animation.args.source.args &&
+          animation.args.source.args.url;
+
+        var isReportable =
+          animation.args &&
+          animation.args.reportable
+
+        return hasSourceUrl && isReportable;
```

Comment:
I'm still not 100% sure how the rest of the code works, so I'm usually much more defensive than usual. That's why I tend to do these kind of things in the interface with the rest of the code.

But if `args.reportable` works, I'd use that. I'll try it out when I add the missing changes for the report generation.